client/components/thank-you-v2/footer/style.scss

```diff
@@ -0,0 +1,67 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.thank-you__details-list {
+	max-width: 650px;
+	margin: 40px auto 30px;
+	width: 100%;
+
+	@include break-mobile {
+		margin-top: 50px;
+	}
+
+	.thank-you__details-list-inner {
+		@include break-small {
+			display: flex;
+			gap: 30px;
+			padding: 0 25px;
+		}
+	}
+
+	.thank-you__detail {
+		margin-bottom: 50px;
+
+		@include break-small {
+			width: 50%;
+		}
+	}
+
+	.thank-you__detail-title {
+		font-size: $font-body;
+		line-height: 20px;
+		font-weight: 500;
+		color: var(--studio-gray-100);
+	}
+
+	.thank-you__detail-description {
+		color: var(--color-text-subtle);
+		font-size: $font-body-small;
+		line-height: 20px;
+		font-weight: 400;
+		margin: 8px 0 16px;
+	}
+
+	.thank-you__detail-button {
```

Comment:
Hi @tyxla!

Thanks for the ping to Dotcom Design. Can you provide screenshots so we can discuss this? Slack could also work.

Reply:
>Might be a good idea to involve @Automattic/dotcom-design folks in reviewing the design for such component and design unification opportunities.

For context, the changes in this PR is part of the effort for unifying the congratulation pages (see pb5gDS-36j-p2 and pb5gDS-38y-p2#comment-4475). This PR is strictly a code refactor, and should not change the design of the page itself.

<img width="1115" alt="Screen Shot 2024-01-10 at 9 02 53 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/451cba6d-0917-4fcc-9084-0c1a1913dfbd">

>I wanted to ensure that we've tried our best to reuse any possible existing components rather than introducing new ones. 

The idea is to eventually replace [`<ThankYou>`](https://github.com/Automattic/wp-calypso/tree/trunk/client/components/thank-you) with `<ThankYouV2>`, which is the redesigned version of the Thank You page.